### DISCUSSION TITLE
After training on my DB, I'm not getting answers related to it.

### DISCUSSION THREAD

**Discussion by AbdoDabbas** [vanna-ai/vanna#630] [Q&A]
First of all I'm very new to this AI thing so please bare with me and consider that I may missed installing something.

I'm using MSSQL to train the model, and chromadb as vector db and llama3.1 (4.7GB size) as a local ai model.

I followed exactly the sample in this page https://vanna.ai/docs/mssql-ollama-chromadb/ , but I faced two issues:
1. When the code reaches the `vn.train` method, it stops, I don't know if training is done or not, but I tries to log some info on cmd after calling the `train` but it doesn't reach it.
2. I thought the training is done, and all, now I commented the `train` line in the code and called the `ask` function, it shows some data and it's trying to call the LLM model to ask it the question, but the model response like it doesn't know what I'm asking for, even when I tried to run the Flask app, and ask the question about how many of X (table name) do we have ? expecting to response with `SELECT Count(*) FROM X` but I get something like "I'm a large language model and I don't have this info ...etc".

BTW: If I used `mistral` as a model instead of `llama3.1`, it starts to understand and make sense of the answers !

Here's my python code for a reference:
```python
from vanna.vannadb.vannadb_vector import VannaDB_VectorStore
from vanna.chromadb.chromadb_vector import ChromaDB_VectorStore
from vanna.ollama import Ollama
import logging

logging.basicConfig(level=logging.INFO,
                    format='%(asctime)s %(levelname)s %(message)s',
                    datefmt='%Y%m%d %H%M%S')
                    
# The Ollama class already knows how to connect to a local ollama service
# The ChromaDB_VectorStore will store data. It will only need to be populated once and persists it.
class MyVanna(ChromaDB_VectorStore, Ollama):
    def __init__(self, config=None):
        ChromaDB_VectorStore.__init__(self, config=config)
        Ollama.__init__(self, config=config)

vn = MyVanna(config={'model': 'llama3.1'})
vn.connect_to_mssql('Driver=SQL Server;Server=ServerName;DATABASE=DBName;')

# The information schema query may need some tweaking depending on your database. This is a good starting point.
df_information_schema = vn.run_sql("SELECT * FROM INFORMATION_SCHEMA.COLUMNS")

# This will break up the information schema into bite-sized chunks that can be referenced by the LLM
plan = vn.get_training_plan_generic(df_information_schema)

vn.train(plan=plan)
 ## Code doesn't reach here, but if I commented the previous row, it continues and runs the flask app.
logging.info('Done training.')


# To run vanna Flask app 
logging.info('running vannaFlaskApp')
from vanna.flask import VannaFlaskApp
app = VannaFlaskApp(vn, allow_llm_to_see_data=True)
app.run()
```

**Comment by RodolfoMontenegro** [comment]
Have you tried adding the crhromadb path?
